src/components/Header/announcement.tsx

```diff
@@ -20,6 +22,10 @@ const ReadMoreLink = styled("a")(
 )
 
 const Announcement = () => {
+  const { pathname } = useLocation()
+  if (pathname.includes("/alpha")) {
```

Comment:
can we be more specific? this excludes e.g. https://deploy-preview-330--scroll-io.netlify.app/blog/alphaTestnet